Makefile

```diff
@@ -0,0 +1,14 @@
+latex_files := $(wildcard *.tex chapters/*.tex config/*.tex appendices/*.tex)
+main_latex_file := zeth-protocol-specification.tex
+references := references.bib
+
+.PHONY: specs clean clean-all
+
+specs: $(main_latex_file) $(latex_files) $(references)
+	latexmk -pdf
```

Comment:
On my machine, without `-halt-on-error` errors cause the process to sit waiting for input.

Reply:
Hum... interesting, I have a different behavior on ubuntu. Seems like this is the right moment to tackle https://github.com/clearmatics/zeth-specifications/issues/5 to add the build on macOS/ubuntu on the CI then

Reply:
Just to clarify, there is no error in the branch.  The proble mis that if there *were* an error, the process halts at a prompt waiting for input.  I personally prefer the process to exit with error, and that's the behaviour we would probably want on the CI, so I'm suggesting we keep the `-halt-on-error` flag here.

(To see what I mean, edit a file to use an invalid commannd `\nosuchcommand` and run `make`)

Reply:
Makes sense @dtebbs - thanks for clarifying. Agreed. I'll add the flag